django_lightweight_queue/app_settings.py

```diff
@@ -1,52 +1,177 @@
+        self._middleware = value
+
+    @property
+    def IGNORE_APPS(self):
+        # Apps to ignore when looking for tasks. Apps must be specified as the dotted
+        # name used in `INSTALLED_APPS`. This is expected to be useful when you need to
+        # have a file called `tasks.py` within an app, but don't want
+        # django-lightweight-queue to import that file.
+        # Note: this _doesn't_ prevent tasks being registered from these apps.
+        if not self._ignore_apps:
+            self._ignore_apps = self._get('IGNORE_APPS', ())
+        return self._ignore_apps  # type: Sequence[str]
+
+    @IGNORE_APPS.setter
+    def IGNORE_APPS(self, value):
+        self._ignore_apps = value
+
+    @property
+    def REDIS_HOST(self):
+        if not self._redis_host:
+            self._redis_host = self._get('REDIS_HOST', '127.0.0.1')
+        return self._redis_host  # type: str
+
+    @REDIS_HOST.setter
+    def REDIS_HOST(self, value):
+        self._redis_host = value
+
+    @property
+    def REDIS_PORT(self):
+        if not self._redis_port:
+            self._redis_port = self._get('REDIS_PORT', 6379)
+        return self._redis_port  # type: int
+
+    @REDIS_PORT.setter
+    def REDIS_PORT(self, value):
+        self._redis_port = value
+
+    @property
+    def REDIS_PASSWORD(self):
+        if not self._redis_password:
+            self._redis_password = self._get('REDIS_PASSWORD', None)
+        return self._redis_password  # type: Optional[str]
+
+    @REDIS_PASSWORD.setter
+    def REDIS_PASSWORD(self, value):
+        self._redis_password = value
+
+    @property
+    def REDIS_PREFIX(self):
+        if not self._redis_prefix:
+            self._redis_prefix = self._get('REDIS_PREFIX', '')
+        return self._redis_prefix  # type: str
+
+    @REDIS_PREFIX.setter
+    def REDIS_PREFIX(self, value):
+        self._redis_prefix = value
+
+    @property
+    def ENABLE_PROMETHEUS(self):
+        if not self._enable_prometheus:
+            self._enable_prometheus = self._get('ENABLE_PROMETHEUS', False)
+        return self._enable_prometheus  # type: bool
+
+    @ENABLE_PROMETHEUS.setter
+    def ENABLE_PROMETHEUS(self, value):
+        self._enable_prometheus = value
+
+    @property
+    def PROMETHEUS_START_PORT(self):
+        # Workers will export metrics on this port, and ports following it
+        if not self._prometheus_start_port:
+            self._prometheus_start_port = self._get('PROMETHEUS_START_PORT', 9300)
+        return self._prometheus_start_port  # type: int
+
+    @PROMETHEUS_START_PORT.setter
+    def PROMETHEUS_START_PORT(self, value):
+        self._prometheus_start_port = value
+
+    @property
+    def ATOMIC_JOBS(self):
+        if not self._atomic_jobs:
+            self._atomic_jobs = self._get('ATOMIC_JOBS', True)
+        return self._atomic_jobs  # type: bool
+
+    @ATOMIC_JOBS.setter
+    def ATOMIC_JOBS(self, value):
+        self._atomic_jobs = value
+
+
+settings = Settings()
```

Comment:
Thoughts on naming this variable `app_settings`? I think that would clarify that these values are specific to DLQ and would mean that there's less diff noise from the updates around the code. (We'd still have the imports changing, but all the usages would be untouched)

Reply:
The only reason I named it settings was because the import would become `from .app_settings import app_settings` which felt a little silly. I'm happy to revert it back to `app_settings` though.